src/reset.css

```
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  padding: 0;
+}

```

```suggestion
body {
  margin: 0
}
```
it will be better not to reset all (`*`) margins and paddings. it will be confusing.

let's specify the elements for which you want to reset the styles